recipe/0001-Fix-for-mamba-seg-fault.patch

```diff
@@ -0,0 +1,44 @@
+From 0b80f87f565abbfdc765569cd045ab49601b8115 Mon Sep 17 00:00:00 2001
+From: Nishidha Panpaliya <npanpa23@in.ibm.com>
+Date: Tue, 4 Apr 2023 22:47:15 -0700
+Subject: [PATCH] Fix for mamba seg fault
+
+---
+ libmamba/src/core/url.cpp | 18 +++++++++++-------
+ 1 file changed, 11 insertions(+), 7 deletions(-)
+
+diff --git a/libmamba/src/core/url.cpp b/libmamba/src/core/url.cpp
+index 8c1ad222..81c15cdd 100644
+--- a/libmamba/src/core/url.cpp
++++ b/libmamba/src/core/url.cpp
+@@ -161,16 +161,20 @@ namespace mamba
+             u = u.substr(0, u.size() - 13);
+         }
+ 
+-        unsigned char hash[16];
+-
+-        EVP_MD_CTX* mdctx;
+-        mdctx = EVP_MD_CTX_create();
+-        EVP_DigestInit_ex(mdctx, EVP_md5(), nullptr);
++        const EVP_MD* type = EVP_md5();
++        unsigned char hash[EVP_MAX_MD_SIZE];
++        if (FIPS_mode() || FIPS_mode_set(1))
```

Comment:
Why do we need `FIPS_mode_set(1)` here?

Reply:
Ahh, copy paste error! :) Will remove it.

Reply:
Great :)

Reply:
Thanks for spotting this. I should have concentrated more in my review...